.github/workflows/deploy-secrets.yml

```diff
@@ -36,6 +36,7 @@ jobs:
       ESDC_RESPONSE_FOLDER: ${{ secrets.ESDC_RESPONSE_FOLDER }}
       ESDC_ENVIRONMENT_CODE: ${{ secrets.ESDC_ENVIRONMENT_CODE }}
       SFAS_RECEIVE_FOLDER: ${{ secrets.SFAS_RECEIVE_FOLDER }}
+      IER_REQUEST_FOLDER: ${{ secrets.IER_REQUEST_FOLDER }}
```

Comment:
A general comment about the "IER_Request" naming to be used, the docs have IER 1 to 12, so should we use "IER" or "IER12"?